src/ui/public/agg_table/agg_table.js

```diff
@@ -13,6 +13,7 @@ define(function (require) {
       template: require('ui/agg_table/agg_table.html'),
       scope: {
         table: '=',
+        exportTitle: '@',
```

Comment:
Does /src/ui/public/agg_table/__tests__/_table.js need to be updated with this attr?  Can the title be tested?